RFC-0039-cuda-support.md

```diff
@@ -0,0 +1,197 @@
+
+# [CUDA version support]
```

Comment:
I think there is some ambiguity below on what it means to "support" a give version of cuda.
In particular make it clear that this is always about full releases, unless specified otherwise.

Reply:
I would also add that we are defining the *PyTorch binary* support. Source builds are most of the time supported at CUDA release.